src/main/java/build/buf/protovalidate/internal/evaluator/MessageValue.java

```diff
@@ -56,4 +56,9 @@ public List<Value> repeatedValue() {
   public Map<Value, Value> mapValue() {
     return Collections.emptyMap();
   }
+
+  @Override
+  public Object raw() {
```

Comment:
Instead of this method, we should be able to call `Value.value(Object.class)` instead.